### PR TITLE
Add html and css file copied from admissions trial project

### PR DESCRIPTION
**Pull request by BranBayou**
From my admissions trial project, I copied html and css files to give code review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
```
Comment (BranBayou) [inline index.html:10]: Please remove all the unnecessary blank lines and spaces. This will make your code look squashed and unreadable.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+            </a>
+          </li>
+        </ul>
+
+    </nav>
+
+    <main id="main-doc">
+
+      <section class="main-section" id="Introduction">
+
```
Comment (BranBayou) [inline index.html:57]: Again, please remove all the unnecessary blank lines and spaces. This will make your code look messy and hard to follow the indentations.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+          </li>
+        </ul>
+
+    </nav>
+
+    <main id="main-doc">
+
+      <section class="main-section" id="Introduction">
+        
+          <header>Introduction</header>
```
Comment (BranBayou) [inline index.html:58]: Please use h1 or h2 tag instead of the header tag. It's a good practice to use a header tag to represent a container for introductory content like navigations and links.

**Review by BranBayou — COMMENTED** [review]
# Changes required :recycle: 

Please check my comments below and make all the necessary changes.

Submit for re-review when you made all the required changes.

Good luck! :+1:

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @BranBayou 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ Good job for suggest the comments polite, 
✔️ Good job for using the GitHub inline features.
✔️ Good Job For Implementing Review your project from the past.


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
